src/components/Capture/Document.js

```diff
@@ -14,6 +14,75 @@ import { localised } from '../../locales'
 import FallbackButton from '../Button/FallbackButton'
 import style from './style.scss'
 
+const LOCALES_MAPPING = {
+  passport: {
+    front: {
+      title: 'doc_submit.title_passport',
+      body: 'photo_upload.body_passport',
+    },
+  },
+  driving_licence: {
+    front: {
+      title: 'doc_submit.title_license_front',
+      body: 'photo_upload.body_license_front',
+    },
+    back: {
+      title: 'doc_submit.title_license_back',
+      body: 'photo_upload.body_license_back',
+    },
+  },
+  national_identity_card: {
+    front: {
+      title: 'doc_submit.title_id_front',
+      body: 'photo_upload.body_id_front',
+    },
+    back: {
+      title: 'doc_submit.title_id_back',
+      body: 'photo_upload.body_id_back',
+    },
+  },
+  residence_permit: {
+    front: {
+      title: 'doc_submit.title_permit_front',
+      body: 'photo_upload.body_permit_front',
+    },
+    back: {
+      title: 'doc_submit.title_permit_back',
+      body: 'photo_upload.body_permit_back',
+    },
+  },
+  bank_building_society_statement: {
+    front: {
+      title: 'doc_submit.title_bank_statement',
+      body: 'photo_upload.body_bank_statement',
+    },
+  },
+  utility_bill: {
+    front: {
+      title: 'doc_submit.title_bill',
+      body: 'photo_upload.body_bill',
+    },
+  },
+  council_tax: {
+    front: {
+      title: 'doc_submit.title_tax_letter',
+      body: 'photo_upload.body_tax_letter',
+    },
+  },
+  benefit_letters: {
+    front: {
+      title: 'doc_submit.title_benefits_letter',
+      body: 'photo_upload.body_benefits_letter',
+    },
+  },
+  government_letter: {
+    front: {
+      title: 'doc_submit.title_government_letter',
+      body: 'photo_upload.body_government_letter',
+    },
+  },
+}
+
```

Comment:
Might be worth putting this mapping in a different file. Maybe it could be in `src/components/Capture/helpers/locales_maps.js` and you could export it as `DOCUMENT_LOCALES_MAPPING`?